### ISSUE TITLE
k8s pod network cidr support

### ISSUE THREAD

**Issue by qianlongzt** [telepresenceio/telepresence#1786]
in macOS and telepresence 2.3.0

in tun mode, i will see  service cidr route like this

```
10.96.0.0/12           10.96.0.1
```
but if i want direct connect to pod (example 10.32.0.0/12), no route to host.

i can do manual `sudo route -n add -net 10.32.0.0/12 10.96.0.0`, it work. but i want telepresence auto support it

**Comment by thallgren** [comment]
And Telepresence will support it when this PR is merged: #1776.

**Comment by efunk** [comment]
Delivered in the Telepresence 2.3.1 GA release.

**Comment by qianlongzt** [comment]
use also-proxy work for me, https://www.telepresence.io/docs/latest/reference/config/#alsoproxy

but i found bug     
```yaml
apiVersion: v1
clusters:
- cluster:
    server: https://xxxxxxxxx:6443
    extensions:
    - name: telepresence.io
      extension:
        also-proxy:
        - 10.32.0.0/12
        - 172.19.39.153/32
```

i got 4 route, should be 2,right ?

10.32/12           10.32.0.1          UCS          utun2
10.32.0.1          10.32.0.0          UH           utun2
172.19.39.1        172.19.39.153      UH           utun2
172.19.39.153/32   172.19.39.1        UCS          utun2

**Comment by thallgren** [comment]
@qianlongzt yes, the subnets covered by other subnets should be removed. This will be fixed when #1801 is released.

**Comment by efunk** [comment]
Resolved in Telepresence 2.3.2 GA
